File: android/src/main/java/ru/vvdev/yamap/RNMarker.java
package ru.vvdev.yamap;

public class RNMarker {
    public double lon;
    public double lat;
    public String id;
    public boolean isSelected;
    public RNMarker(double _lon, double _lat, String _id, boolean _isSelected) {
        lon = _lon;
        lat = _lat;
        id = _id;
        isSelected = _isSelected;
    }
}

File: ios/RNMarker.m
#import "RNMarker.h"


@implementation RNMarker {
}
- (RNMarker*)initWithJson:(NSDictionary *)json {
    self.lon = [[json valueForKey:@"lon"] doubleValue];
    self.lat = [[json valueForKey:@"lat"] doubleValue];
    self._id = [json valueForKey:@"id"];
    self.isSelected = [[json valueForKey:@"selected"] boolValue];
    return self;
}

@end
